charts/invenio/templates/web-deployment.yaml

```
@@ -37,6 +37,9 @@ spec:
           value: {{ include "invenio.rabbitmq.uri" . }}
         - name: INVENIO_SQLALCHEMY_DATABASE_URI
           value: {{ include "invenio.sqlAlchemyDbUri" . }}
+        {{- if .Values.web.extraEnvVars }}
+        {{- .Values.web.extraEnvVars | toYaml | nindent 8 }}
+        {{- end }}
```

To reduce code duplication, I prefer using `with` statement:

```suggestion
        {{- with .Values.web.extraEnvVars }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```

But if-statements like this are clearly valid too, so I think we can go ahead with this. Just wanted to mention it. 🙂